pilot/pkg/networking/core/v1alpha3/extension_config_builder.go

```diff
@@ -29,7 +30,10 @@ func (configgen *ConfigGeneratorImpl) BuildExtensionConfiguration(
 ) []*core.TypedExtensionConfig {
 	envoyFilterPatches := push.EnvoyFilters(proxy)
 	extensions := envoyfilter.InsertedExtensionConfigurations(envoyFilterPatches, extensionConfigNames)
+
 	wasmPlugins := push.WasmPlugins(proxy)
```

Comment:
Why didn't we implement a Telemetry push method similar to this?

Reply:
https://istio.io/latest/docs/reference/config/telemetry/#AccessLogging-Filter cannot send with ECDS.